EXPO/screens/UserInfoDBManager.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import  SQLite   from 'react-native-sqlite-storage';
+import { localDB } from '../screens/Constants';
+
+
+export default  class UserInfoDBManager extends Component {
+
+  constructor() {
+       this.sqlite = SQLite;
+       this.sqlite.DEBUG(true);
+       this.sqlite.enablePromise(true);
+       this.sqlite.openDatabase({
+           name: localDB.dbName,
+           location: "1"
+       }).then((db) => {
+           this.dbInstance = db;
+       })
+   }
+
+   createTable() {
+     return new Promise((resolve, reject) => {
+            this.dbInstance.executeSql('CREATE TABLE IF NOT EXISTS ' + localDB.tableName.tblLogin + ' (id  INTEGER PRIMARY KEY ,firstName TEXT,lastName TEXT,avator:TEXT)'
```

Comment:
what is this.dbInstance is undefined ? this is the case where DB connection failed.